osquery/killswitch/killswitch_refreshable_plugin.cpp

```diff
@@ -0,0 +1,53 @@
+#include <osquery/dispatcher.h>
+#include <osquery/flags.h>
+#include <osquery/killswitch.h>
+#include <osquery/killswitch/killswitch_refreshable_plugin.h>
+
+namespace osquery {
+
+FLAG(uint32,
+     killswitch_refresh_rate,
+     10,
+     "Refresh rate of killswitch in seconds");
+
+class KillswitchRefresher : public InternalRunnable {
+ public:
+  KillswitchRefresher(size_t update_interval)
+      : InternalRunnable("KillswitchRefreshRunner"),
+        update_interval_(update_interval) {}
+  /// A simple wait/interruptible lock.
+  void start() override {
+    while (!interrupted()) {
+      osquery::Killswitch::get().refresh();
+      pauseMilli(
+          std::chrono::milliseconds(std::chrono::seconds(update_interval_)));
+    }
+  }
+
+ private:
+  const size_t update_interval_;
```

Comment:
The `std::size_t` is a special type for memory related arithmetics and container indexes [proof](https://en.cppreference.com/w/cpp/types/size_t). Why do you use it for the time interval? Why not a `std::duration`?